report/finished/chapters/appendix.tex

```diff
@@ -57,8 +57,8 @@ \section*{Project structure}
 \end{itemize}
 
 \section*{Meetings}
-Every week the group will be meeting on Mondays and Tuesdays to catch up and support each other. This scheme may change in the future if needed. 
-The Monday meetings will have the following agenda where the goal is to catch up with the whole project group and discuss the project
+Every week the group met on Mondays and Thursdays to catch up and support each other. This structure gave the students a great deal of responsibility to do work for each meeting while still maintaining a good structure of the project and encouraged discussions. 
+The Monday meetings had the following agenda where the goal was to catch up with the whole project group and discuss the project and the weekly seminar
```

Comment:
"The agenda of the monday meeting was to catch up with the whole project group, discuss the project and prepare for the weekly seminar. Topics that was brought up during these meeting were:"  - flows a bit better and is less confusing in my opinion